packages/admin/cms-admin/src/dam/DataGrid/DamContextMenu.tsx

```diff
@@ -68,6 +72,14 @@ const FolderInnerMenu = ({ folder, openMoveDialog }: FolderInnerMenuProps): Reac
                     >
                         <FormattedMessage id="comet.pages.dam.rename" defaultMessage="Rename" />
                     </RowActionsItem>
+                    <RowActionsItem<"a">
```

Comment:
Can't this be inferred?

Reply:
from the component props?

Reply:
Yes

Reply:
Can you show me where and how MUI does it?

Reply:
MUI does it like this:

```tsx
<Typography  component="a" href="..." />
```

`href` and other link props are automatically inferred, there's no need to specify it in the component like this:

```tsx
<Typography<"a">  component="a" href="..." />
```

Reply:
I tried a long time again on friday. Don't know if it's worth to put more work into this.

Reply:
Alright, let's leave it like this. Maybe we'll need to fix it in the future